packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionInvoke.ts

```diff
@@ -71,7 +71,8 @@ export async function forceFunctionDebugInvoke(sourceUri: Uri) {
       await FunctionService.instance.stopDebuggingFunction(localRoot);
       telemetryService.sendCommandEvent(
         'force_function_debug_invoke',
-        startTime
+        startTime,
+        { language: FunctionService.instance.getFunctionLanguage() }
```

Comment:
This change is fine for the scope of the PR, but just wanted to FYI that by having `sendCommandEvent` here, I believe we are sending two events for the same execution since the `execute()` method of `SfdxCommandletExecutor` is already calling this through `logMetric`. This could report double the executions in insights

Reply:
@brpowell would you recommend another WI to make a similar change as we did in forceFunctionCreate? I think that would also apply to forceFunctionStop.

Reply:
That would be worth filing if that is the case, otherwise we're getting incorrect data on command usage.

Reply:
Hello 👋 , intern here. sorry to interject but I had a question about this. I'm not entirely sure what a WI is, but regarding the metric logging *I think* my [PR for switching function invocation to using a library instead of CLI](https://github.com/forcedotcom/salesforcedx-vscode/pull/3282) handles that metric. So maybe it's worth holding on that improvement for now? I'm not entirely sure, just figured I'd make note of it

Reply:
Thanks for the heads up! If you get the sense that it's logging dupes and you have a chance to change it in your PR, feel free to make a change or note of it there, factoring in these changes too. Sounds like it's a good change if it gets us further off of the CLI!